conversion_helpers.go

```diff
@@ -14,6 +14,45 @@ import (
 //   - conversion function from an external type to an internal type
 // ****************************************************************************
 
+// PayloadStringer can be used to convert payload bytes to string to show in web ui.
+type PayloadStringer interface {
```

Comment:
nit: Can we rename this to `BytesStringer` or something more general? (Context: I'm currently working on https://github.com/hibiken/asynq/issues/265 and we'll be able to store result data as `[]byte` and I think users may want to use this interface to display the result data in the web UI as well)